package.json

```diff
@@ -22,10 +22,10 @@
     "lint:watch": "esw --watch --config ./eslint/eslintrc packages",
     "lint": "eslint --config .eslintrc.js packages",
     "publish": "node utils/pre-publish-check.js && git diff --stat --exit-code HEAD && yarn build:cjs && yarn build:es6 && lerna publish",
-    "start:prod": "yarn run build:styleguidist && yarn run styleguidist --config styleguide.prod.config.js",
-    "start-storybook": "start-storybook -s ./static",
+    "start:prod": "yarn run build:styleguidist && yarn run start:styleguidist --config styleguide.prod.config.js",
+    "start:storybook": "start-storybook -s ./static",
```

Comment:
I used a `-` here because I was confused one time when I was trying to run `yarn start-storybook` and the static assets weren't showing up.  I'll just have to remember to use `:`.

Reply:
I changed it for consistency with how we've done everything else. No strong opinion.